spec/models/hyrax/collection_type_spec.rb

```diff
@@ -194,4 +194,13 @@
       end
     end
   end
+
+  describe "collection types_for" do
+    let(:user_collection_type) { create(:user_collection_type) }
+    let(:user) { create(:user) }
+
+    it 'returns types of collections user is authorized to create' do
+      expect(Hyrax::CollectionType.collection_types_for(user)).to eq(Hyrax::CollectionType.all)
```

Comment:
When this is no longer stubbed out, should this test still expect all types?  If not, can you add a TODO that this needs to be updated when the method is completed?